src/map/swipe.js

```diff
@@ -0,0 +1,149 @@
+import angular from 'angular';
+import * as olEvents from 'ol/events.js';
+import '../../examples/mapswipe.css';
```

Comment:
I hadn't noticed that one.

Remove this.  This should be imported in the example itself, not in the component.